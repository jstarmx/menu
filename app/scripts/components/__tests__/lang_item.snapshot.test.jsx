import React from 'react';
import renderer from 'react-test-renderer';

import LangItem from '../lang_item';

it('renders correctly', () => {
  const tree = renderer.create(
    <LangItem code="it" name="Italiano" selected onClick={ () => {} } />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
