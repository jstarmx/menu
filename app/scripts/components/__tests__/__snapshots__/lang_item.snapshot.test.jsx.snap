// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="lang-popup__item lang-popup__item--active"
>
  <button
    className="lang-popup__button"
    onClick={[Function]}
  >
    <svg
      className="flag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<use xlink:href=\\"#it\\"></use>",
        }
      }
    />
    <span
      className="lang-popup__label"
    >
      Italiano
    </span>
  </button>
</li>
`;
